src/compas_rhino/scene/boxobject.py

```diff
@@ -46,10 +46,8 @@ def draw(self, color=None):
         attr = attributes(name=self.geometry.name, color=color, layer=self.layer)
 
         geometry = box_to_rhino(self.geometry)
-
-        if self.transformation:
-            transformation = transformation_to_rhino(self.transformation)
-            geometry.Transform(transformation)
+        transformation = transformation_to_rhino(self.transformation_world)
+        geometry.Transform(transformation)
```

Comment:
Now in implementation of each objects, we should always use `transformation_world`.